float32 flex_myobrick_pwm
float32 extend_myobrick_pwm
---
bool success